pyresample/test/test_spherical.py

```diff
@@ -117,7 +128,7 @@ def test_distance(self):
     def test_hdistance(self):
         """Test Haversine formula."""
         d = SCoordinate(0, 0).hdistance(SCoordinate(1, 1))
-        self.assertTrue(np.allclose(d, 1.2745557823062943))
+        assert (np.allclose(d, 1.2745557823062943))
```

Comment:
```suggestion
        np.testing.assert_allclose(d, 1.2745557823062943)
```

Reply:
To save me time...could you do this same update to all your use cases in the tests and remove the unnecessary surrounding `()` in your assert statements?